pegasus/test/test_string.rb

```diff
@@ -25,8 +25,8 @@ def test_multiply_concat
   end
 
   def test_to_bool
-    ['true','t','yes','y','1'].each{|true_value| assert true_value.to_bool}
-    ['', 'false','f','no','n','0'].each{|false_value| assert !false_value.to_bool}
+    %w(true t yes y 1).each{|true_value| assert true_value.to_bool}
+    ['', 'false', 'f', 'no', 'n', '0'].each{|false_value| assert !false_value.to_bool}
```

Comment:
Oops. That was me playing around with the five threshold and doing a poor job reverting, will revert the spacing change.